scripts/globals/job_utils/rune_fencer.lua

```diff
@@ -0,0 +1,54 @@
+-----------------------------------
+-- Rune Fencer Job Utilities
+-----------------------------------
+require("scripts/settings/main")
+require("scripts/globals/ability")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+require("scripts/globals/weaponskills")
+require("scripts/globals/jobpoints")
+-----------------------------------
+xi = xi or {}
+xi.job_utils = xi.job_utils or {}
+xi.job_utils.rune_fencer = xi.job_utils.rune_fencer or {}
+-----------------------------------
+
+local function getRUNLevel(player)
+    return player:getMainJob() == xi.job.RUN and player:getMainLvl() or player:getSubLvl()
+end
+
+local function applyRuneEnhancement(effectType, player)
+    local RUNLevel = getRUNLevel(player)
+    local meritBonus = (player:getMerit(xi.merit.MERIT_RUNE_ENHANCE) * 2) -- 2 more elemental resistance per merit for a maximum total of (2*5) = 10
```

Comment:
Line 22 - This parenthesis arround "getMerit * 2" is unnecesary.
Also
`require("scripts/settings/main")`
I dont see any actual setting being used in this script, so it's unnecesary.

Reply:
I realized after the fact when actually trying to implement the merit for Rune Enchantment that you can set the power per level in the merits.sql file so the multiplication isn't necessary at all... talk about a micro-optimization! Will fix when I get home.